packages/react/src/metadata.tsx

```diff
@@ -385,6 +385,54 @@ const acceptedFieldTypes = new Set([
   FieldType.HasOne,
 ]);
 
+export const filterAutoTableFieldList = (fields: FieldMetadata[] | undefined, options?: { include?: string[] }) => {
+  if (!fields) {
+    return [];
+  }
+
+  let subset = fields;
+
+  if (options?.include) {
+    // When including fields, the order will match the order of the `include` array
+    subset = [];
+    const includes = new Set(options.include);
+
+    for (const includedFieldApiId of Array.from(includes)) {
+      const metadataField = fields.find((field) => field.apiIdentifier === includedFieldApiId);
+      if (metadataField) {
+        subset.push(metadataField);
+      }
```

Comment:
![CleanShot 2024-07-18 at 15 28 32@2x](https://github.com/user-attachments/assets/6ab6c9c1-0c76-4a7a-af60-fab5939b2a67)

When given a column name that does not exist, we get nothing rendered. Perhaps we should have a `else throw Error()` here so that the user knows why there is nothing rendered

Reply:
It's because it makes the GraphQL request to respond to error 400.
<img width="683" alt="CleanShot 2024-07-18 at 15 32 09@2x" src="https://github.com/user-attachments/assets/4fafdc5b-4d4b-44ed-90c9-8c01aa694f98">

I think maybe instead we should show a banner similar to the form.

Reply:
Good point, a banner would be better so that the user can have a dynamically changing list of columns without stress of getting a whole exception